src/roswire/common/package.py

```diff
@@ -221,7 +239,7 @@ def __iter__(self) -> Iterator[str]:
 
     @staticmethod
     def from_dict(d: List[Dict[str, Any]]) -> 'PackageDatabase':
-        return PackageDatabase([Package.from_dict(dd) for dd in d])
+        return PackageDatabase([Package.from_dict(dd) for dd in d], [])
```

Comment:
If you go with the optimisation suggested above, you could drop the list comprehension (for the first argument) and do `( ... )` instead to use a generator rather than a list.